samples/Cryptocash.Api/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Meta.BankNoteMetadata.g.cs

```diff
@@ -48,4 +49,22 @@ public static Nox.Types.Text CreateCashNote(System.String value)
         public static Nox.Types.Money CreateValue(IMoney value)
             => Nox.Types.Money.From(value);
         
+
+        /// <summary>
+        /// User Interface for property 'CashNote'
+        /// </summary>
+        public TypeUserInterface? CashNoteUserInterface(NoxSolution solution) 

```

Comment:
now looking to the name.... @DanielHodgeGH  do y have a better suggestion?

CashNoteUiSettings
CashNoteUiOptions
CashNoteUi
:(